index.html

```diff
@@ -7,13 +7,62 @@
     <!-- Here we import the Boostrap CSS stylesheet -->
     <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-giJF6kkoqNQ00vy+HMDP7azOuL0xtbfIcaT9wjKHr8RbDVddVHyTfAAsrekwKmP1" crossorigin="anonymous">
     <!-- Add a link to your custom CSS here -->
-
-    <title>Responsive grid project</title>
+    <link rel="stylesheet" href="./css/main.css">
+    <title>Domenico DC - Responsive grid project</title>
 </head>
 
 <body>
+    <header>
+        <section class="flex-nav">
+            <h5><strong>Domenico Delli Carri</strong> / FULL STACK DEVELOPER</h5>
+            <nav>
+                <a href="./index.html">HOME</a>
+                <a href="">RESUME</a>
+                <a href="./projects.html">PROJECTS</a>
+                <a href="./contact.html">CONTACT</a>
+            </nav>
+        </section>
+    </header>
+
+
+    <main>
+        <div class="container container-class">
+            <div class="row">
+                <div class="col-md-5 col-12 background-beige">
```

Comment:
See my comment about the `background-beige` class (it shouldn't include a margin) but otherwise this is a good use of Boostrap classes. 
You've got a `container`, then a `row`, then `col-....` classes so all good here 👍